docs/content/en/docs/getting-started.md

```diff
@@ -0,0 +1,488 @@
+---
+title: Getting started with KLT
+description: Get started with the Keptn Lifecycle Toolkit
+weight: 10
+hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
+---
+
+Keptn Lifecycle Toolkit works whether or not you use a GitOps strategy.
+The following is an imperative walkthrough.
+
+## Prerequisites
+
+- [Docker](https://docker.com)
+- [kubectl](https://kubernetes.io/docs/reference/kubectl/)
+- A Kubernetes cluster >= 1.24 (we recommend [Kubernetes kind](https://kind.sigs.k8s.io/docs/user/quick-start/))
+  (`kind create cluster`)
+- [Helm](https://helm.io)
+
+## Objectives
+
+- Install Keptn Lifecycle Toolkit on your cluster
+- Annotate a namespace and deployment to enable Keptn Lifecycle Toolkit
+- Install Grafana and Observability tooling to view DORA metrics and OpenTelemetry traces
+
+## System Overview
+
+By the end of this page, here is what will be built.
+The system will be built in stages.
+
+![system overview](/docs/assets/install01.png)
+
+## The Basics: A Deployment, Keptn and DORA Metrics
+
+To begin our exploration of the Keptn observability features, we will:
+
+- Deploy a simple application called `keptndemo`.
+
+Keptn will monitor the deployment and generate:
+
+- An OpenTelemetry trace per deployment
+- DORA metrics
+
+![the basics](/docs/assets/install02.png)
+
+Notice though that the metrics and traces have nowhere to go.
+That will be fixed in a subsequent step.
+
+## Step 1: Install Keptn Lifecycle Toolkit
+
+Install the Keptn Lifecycle Toolkit using Helm:
+
+```shell
+helm repo add klt https://charts.lifecycle.keptn.sh
+helm repo update
+helm upgrade --install keptn klt/klt -n keptn-lifecycle-toolkit-system --create-namespace --wait
+```
+
+Keptn will need to know where to send OpenTelemetry traces.
+Of course, Jaeger is not yet installed so traces have nowhere to go (yet),
+but creating this configuration now means the system is preconfigured.
+
+Save this file as `collectorconfig.yaml`.
+It doesn't matter where this file is located on your local machine:
+
+```yaml
+---
+apiVersion: options.keptn.sh/v1alpha1
+kind: KeptnConfig
+metadata:
+  name: keptnconfig-sample
+  namespace: keptn-lifecycle-toolkit-system
+spec:
+  OTelCollectorUrl: 'jaeger-collector.keptn-lifecycle-toolkit-system.svc.cluster.local:4317'
+  keptnAppCreationRequestTimeoutSeconds: 30
+```
+
+Apply the file and restart KLT to pick up the new config:
```

Comment:
@agardnerIT did you also try to apply the `keptnconfig-sample` without restarting the deployments? The restart should not be required, and the new config should be applied on the fly within a couple of seconds. If that is not the case we need to create a bug issue and investigate this further

Reply:
i agree, let's move this to a follow-up